lldb/include/lldb/API/SBModule.h

```diff
@@ -287,6 +287,14 @@ class LLDB_API SBModule {
   ///     A const reference to the file specification object.
   lldb::SBFileSpec GetSymbolFileSpec() const;
 
+  /// Get a list of filespecs associated with all the separate symbol files
+  /// associated with this module.
+  ///
+  /// \return
+  ///     A list of filespecs associated with all the separate symbol files
+  ///     associated with this module.
+  lldb::SBFileSpecList GetSeparateDebugInfoFiles();
```

Comment:
I would prefer a `lldb::SBModuleSpecList` since DWARF in .o files can exist in .a files and `lldb::SBModuleSpec` can fully describe files within files for the .o file in a .a file case.